src/itest/kotlin/nl/info/zac/itest/BpmnDocumentCreationRestServiceTest.kt

```diff
@@ -0,0 +1,289 @@
+                        "taskId" to taskId,
+                        "informatieobjecttypeUuid" to INFORMATIE_OBJECT_TYPE_BIJLAGE_UUID,
+                        "smartDocumentsTemplateGroupName" to SMART_DOCUMENTS_ROOT_GROUP_NAME,
+                        "smartDocumentsTemplateName" to SMART_DOCUMENTS_ROOT_TEMPLATE_1_NAME,
+                        "title" to SMART_DOCUMENTS_FILE_TITLE,
+                        "creationDate" to ZonedDateTime.now(),
+                        "author" to TEST_USER_1_NAME
+                    )
+                ).toString()
+            )
+            Then("the response should be OK and the response should contain a redirect URL to SmartDocuments") {
+                val responseBody = response.body!!.string()
+                logger.info { "Response: $responseBody" }
+                response.code shouldBe HTTP_OK
+
+                with(responseBody) {
+                    shouldContainJsonKeyValue(
+                        "redirectURL",
+                        "$SMART_DOCUMENTS_MOCK_BASE_URI/smartdocuments/wizard?ticket=fakeSmartdocumentsTicketID"
+                    )
+                }
+            }
+        }
+
+        When("the create document attended ('wizard') endpoint is called with all parameters") {
+            val endpointUrl = "$ZAC_API_URI/document-creation/create-document-attended"
+            logger.info { "Calling $endpointUrl endpoint" }
+            val response = itestHttpClient.performJSONPostRequest(
+                url = endpointUrl,
+                requestBodyAsString = JSONObject(
+                    mapOf(
+                        "zaakUuid" to bpmnZaakUuid,
+                        "taskId" to taskId,
+                        "informatieobjecttypeUuid" to INFORMATIE_OBJECT_TYPE_BIJLAGE_UUID,
+                        "smartDocumentsTemplateGroupName" to SMART_DOCUMENTS_ROOT_GROUP_NAME,
+                        "smartDocumentsTemplateName" to SMART_DOCUMENTS_ROOT_TEMPLATE_1_NAME,
+                        "title" to SMART_DOCUMENTS_FILE_TITLE,
+                        "description" to "document description",
+                        "author" to TEST_USER_1_NAME,
+                        "creationDate" to ZonedDateTime.now()
+                    )
+                ).toString()
+            )
+            Then("the response should be OK and the response should contain a redirect URL to Smartdocuments") {
+                val responseBody = response.body!!.string()
+                logger.info { "Response: $responseBody" }
+                response.code shouldBe HTTP_OK
+
+                with(responseBody) {
+                    shouldContainJsonKeyValue(
+                        "redirectURL",
+                        "$SMART_DOCUMENTS_MOCK_BASE_URI/smartdocuments/wizard?ticket=fakeSmartdocumentsTicketID"
+                    )
+                }
+            }
+        }
+
+        When("the create document attended ('wizard') endpoint is called without mandatory parameter") {
```

Comment:
Not too sure if we should test invalid requests in our itests. If we start doing this I'm afraid our itests will explode a bit. At the moment we mostly test happy paths only.

Reply:
Removed